src/test/unit/controls/Bar/BarPanning-spec.js

```diff
@@ -157,4 +157,54 @@ describe("Bar - Panning", () => {
             });
         });
     });
+    describe("On click of panning button", () => {
+        beforeEach(() => {
+            const axisData = utils.deepClone(getAxes(axisTimeSeries));
+            axisData.dateline = [
+                {
+                    showDatelineIndicator: true,
+                    label: {
+                        display: "Release A"
+                    },
+                    color: COLORS.GREEN,
+                    shape: SHAPES.SQUARE,
+                    value: "2016-06-03T12:00:00Z"
+                }
+            ];
+            axisData.pan = { enabled: false };
+            const input = getInput([], false, false);
+            graphDefault = new Graph(axisData);
+            graphDefault.loadContent(new Bar(input));
+        });
+        it("No Data View is removed when legend hold values", () => {
```

Comment:
nit: It can be updated all related test cases
```js
describe("when legend hold values", () => {
     it("should remove the No Data View", () => {

    });
});
``

Reply:
Updated [here](https://github.com/cerner/carbon-graphs/pull/289/commits/3ce6118bc0b5434bd01afa4df0228f42caa0d8c6)